packages/react-docs-example/ts/json/0.1.12.json

```diff
@@ -2288,7 +2288,7 @@
                                                     },
                                                     {
                                                         "type": "reference",
-                                                        "name": "Web3.BlockParam"
+                                                        "name": "BlockParam"
```

Comment:
Don't update these. They are frozen to what `web3-wrapper` looked like at these earlier versions. They can stay unchanged.